src/main/java/org/quiltmc/loader/api/plugin/ModMetadataExt.java

```diff
@@ -54,9 +54,7 @@ default ModLoadType loadType() {
 	}
 
 	@Nullable
-	default ModPlugin plugin() {
-		return null;
-	}
+	ModPlugin plugin();
```

Comment:
I'd rather this was null by default to be honest actually, since I'm not sure how stable plugins-loading-plugins will end up being?